src/core/bitop.d

```diff
@@ -72,20 +72,18 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
+{
 /// ditto
+int bsf(ulong v) pure;
+}
+else version(D_LP64) { } else
```

Comment:
Please just make this `else version(D_LP32)`. My implementation is not good for anything other than 32-bit.